File: KingdomCodeTool/wwwroot/Download/202410291354/Angular/shared/CongViec.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { CongViec } from 'src/app/shared/CongViec.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class CongViecService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "CongViec";
    }
}

File: KingdomCodeTool/wwwroot/Download/202410291449/Service/Implement/CongViecService.cs
namespace Service.Implement
{
    public class CongViecService : BaseService<CongViec, ICongViecRepository>
    , ICongViecService
    {
    private readonly ICongViecRepository _CongViecRepository;
    public CongViecService(ICongViecRepository CongViecRepository) : base(CongViecRepository)
    {
    _CongViecRepository = CongViecRepository;
    }
    }
    }

